src/tiles/expandable-tile.component.ts

```diff
@@ -22,11 +22,11 @@ export interface ExpandableTileTranslations {
 			type="button"
 			(click)="onClick()">
 			<div [attr.aria-label]="(expanded ? collapse : expand).subject | async" class="bx--tile__chevron">
-				<svg *ngIf="!expanded" width="12" height="7" viewBox="0 0 12 7" role="img">
+				<svg *ngIf="!expanded" width="12" height="7" viewBox="0 0 12 7" title="{{expand.subject | async}}" role="img">
 					<title>{{expand.subject | async}}</title>
 					<path fill-rule="nonzero" d="M6.002 5.55L11.27 0l.726.685L6.003 7 0 .685.726 0z"/>
 				</svg>
-				<svg *ngIf="expanded" width="12" height="7" viewBox="0 0 12 7" role="img">
+				<svg *ngIf="expanded" width="12" height="7" viewBox="0 0 12 7" title="{{expand.subject | async}}" role="img">
```

Comment:
We'll want the "collapse" text for this

```suggestion
				<svg *ngIf="expanded" width="12" height="7" viewBox="0 0 12 7" title="{{collapse.subject | async}}" role="img">
```